src/streams.c

```diff
@@ -98,6 +98,67 @@ Int READ_COMMAND ( void )
     return 1;
 }
 
+Obj FuncREAD_ALL_COMMANDS( Obj self, Obj stream, Obj echo )
+{
+    ExecStatus status;
+    Int result_count, result_capacity;
+    Obj result, result_list;
+
+    // 16 Results should be enough for everyone
```

Comment:
that comment scared me, but it turns out that later on, you actually grow the list if necessary... I guess the comment is meant to be snarky ("640kb is enough for everyone" ?) but perhaps here I'd avoid that.